### PR TITLE
T39353 Updated common sections

### PR DESCRIPTION
**Pull request by vikram-bitnami**
This creates common sections and reuses them for charts as per discussion in T39353#937304

### CODE REVIEW

**Comment by vikram-bitnami** [comment]
FYI @raquel-campuzano as these sections can be reused for new docs.

File: common/_upgrade_helm3.md.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <%
 =begin
-apps: all
+apps: airflow,apache,aspnet-core,cassandra,common,consul,contour,discourse,dokuwiki,drupal,ejbca,elasticsearch,etcd,external-dns,fluentd,ghost,grafana,harbor,influxdb,jasperreports,jenkins,joomla,kafka,keycloak,kibana,kong,kube-prometheus,kube-state-metrics,kubewatch,logstash,magento,mariadb, mariadb-galers,mediawiki,memcached,metallb,metrics-server,minio,mongodb,mongodb-shared,moodle,mxnet,mysql,nats,nginx,nginx-ingress-controller,node,node-exporter,odoo,opencart,orangehrm,osclass,owncloud,parse,phabricator,phpbb,phpmyadmin,postgresql,postgresql-ha,prestashop,pytorch,rabbitmq,redis,redis-cluster,redmine,spark,spring-cloud-dataflow,suitecrm,tensorflow-resnet,testlink,thanos,tomcat,wavefront,wildfly,wordpress,zookeeper
```
Comment (vikram-bitnami) [inline common/_upgrade_helm3.md.erb:3]: Updated only for charts that have the Helm 3 section in README

File: common/_configure_ingress.md.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<%
+=begin
+apps: airflow,apache,aspnet-core,consul,discourse,dokuwiki,drupal,ejbca,fluentd,ghost,grafana,grafana-operator,harbor,influxdb,jasperreports,jenkins,joomla,keycloak,kibana,kong,kubeapps,logstash,magento,mediawiki,minio,moodle,nats,nginx,node,odoo,opencart,orangehrm,osclass,owncloud,parse,phabricator,phpbb,phpmyadmin,prestashop,rabbitmq,redmine,solr,spark,suitecrm,testlink,tomcat,wildfly,wordpress
```
Comment (vikram-bitnami) [inline common/_configure_ingress.md.erb:3]: Updated only for charts that have the ingress parameter in README

File: common/_upgrade_helm3.md.erb (hunk 1 above)
Comment (beltran-rubo) [reply common/_upgrade_helm3.md.erb:3]: All the catalog now requires Helm 3, we can enable for all of them

**Review by beltran-rubo — COMMENTED** [review]
the rest LGTM thanks

**Pull request merged**